rest-api-spec/src/main/resources/rest-api-spec/api/bulk.json

```diff
@@ -74,6 +74,14 @@
       "require_alias": {
         "type": "boolean",
         "description": "Sets require_alias for all incoming documents. Defaults to unset (false)"
+      },
+      "maximum_batch_size": {
```

Comment:
1. Why did we choose to call this "maximum_batch_size" and not just "size"?
2. Are there other options that spell out "maximum", I see things like "max_content_length", abbreviated "max".

Reply:
Seems that `maximum_batch_size` is quite confusing and `batch_size` makes more sense. I'll rename. 
You are right about the `max_` prefix, I do see many parameters have `max_`.